lib/Syntax/Trivia.cpp

```diff
@@ -101,6 +104,7 @@ void TriviaPiece::accumulateAbsolutePosition(AbsolutePosition &Pos) const {
     break;
   case TriviaKind::Newline:
   case TriviaKind::CarriageReturn:
+  case TriviaKind::CarriageReturnLineFeed:
     Pos.addNewlines(Count);
```

Comment:
`addNewlines(Count)` only increments `Pos.Offset` by `Count`.
We also should fix `addText(Str)` to take `\r` and `\r\n` into account
see: https://github.com/apple/swift/blob/3f0e7029c5d09ffa2a9daf54e9715bcc068b420e/include/swift/Syntax/RawSyntax.h#L139-L162

Reply:
I will read around there.